### PR TITLE
WIP: Add Attributes API to release notes

### PR DESCRIPTION
**Pull request by vipulnsward**
Still working over it.
r? @sgrif

### CODE REVIEW

File: guides/source/5_0_release_notes.md [hunk 1]
```diff
@@ -48,7 +48,124 @@ ToDo...
...
+  attribute :price_in_cents, :integer # custom type
+  attribute :my_string, :string, default: "new default" # default value
+  attribute :my_default_proc, :datetime, default: -> { Time.now } # default value
+  attribute :field_without_db_column, :integer, array: true 
+end
+
+# after
+store_listing.price_in_cents # => 10
+StoreListing.new.my_string # => "new default"
+Product.new.my_default_proc # => 2015-05-30 11:04:48 -0600
```
Comment (maclover7) [inline guides/source/5_0_release_notes.md]: `StoreListing.new`

File: guides/source/5_0_release_notes.md [hunk 2]
```diff
@@ -48,7 +48,124 @@ ToDo...
...
+model = MyModel.new(field_without_db_column: ["1", "2", "3"])
```
Comment (maclover7) [inline guides/source/5_0_release_notes.md]: `StoreListing.new`

File: guides/source/5_0_release_notes.md [hunk 3]
```diff
@@ -48,7 +48,124 @@ ToDo...
...
+Product.where(price_in_bitcoins: Money.new(5, "GBP"))
+# => SELECT * FROM products WHERE price_in_bitcoins = 0.03412
+```
+
+**Dirty Tracking:**
+
+The type of an attribute is given the opportunity to change how dirty
+tracking is performed. 
+      
+See its
```
Comment (maclover7) [inline guides/source/5_0_release_notes.md]: We should link to `api.`, not `edgeapi.` :grimacing:

**Comment by maclover7** [comment]
Overall looks pretty good :+1:

**Comment by sgrif** [comment]
Do we need this much detail in the release notes? It seems like this includes basically all of the documentation for the API.

**Comment by vipulnsward** [comment]
@sgrif no we don't. I need to condense 2 parts.

**Comment by vipulnsward** [comment]
review @sgrif .
Let me know if something needs to be dropped. This is following convention from previous changelog items.

**Comment by sgrif** [comment]
:+1:

**Comment by vipulnsward** [comment]
Thanks for the review.

**Pull request merged**